src/main/java/com/github/junit5docker/Port.java

```diff
@@ -1,7 +1,21 @@
 package com.github.junit5docker;
 
+/**
+ * Describes a port binding to send to docker.
+ *
+ * @since 1.0
+ */
 public @interface Port {
+
+    /**
+     * @return the port number exposed on localhost.
```

Comment:
Right ! It the socket that is local (i.e on localhost) but the real host can be distant !
My bad :)